src/components/PhotoListForm/PhotoListForm.tsx

```diff
@@ -79,51 +155,43 @@ const PhotoListForm: React.FC<WeddingData> = ({guests, changeView}) => {
             value={description}
             onChange={event => setDescription(event.target.value)}
           />
+          { isLoading ? "loading" :
           <section className="checkboxSection">
             <h3 className="sectionHeader">Guests:</h3>
-            {guestsOptions.map((guest: any, i: number) => {
-              return (
-                <Checkbox
-                  key={i +1}
-                  toggleCheckMark={toggleCheckMark}
-                  {...guest}
-                />
-              )
-              })
-            }
-          </section>
-            {isError && errorMessage}
-          
+            {displayGuests()}
+          </section> }
+          {isError && errorMessage}
           <StyledButton onClick={event => submitPhoto(event)}>
             <div id="translate"></div>
-            <a className="link" id="addListButton">Submit Photo</a>
+            <h3 className="link" id="addListButton">Submit Photo</h3>
```

Comment:
I was wondering what this `<a>` tag error message was about! It makes sense that they shouldn't be used since `a` tags should only be to link to pages outside our website, right?